pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -100,6 +101,65 @@ def test_rpm(self):
         )
         self.assertDictEqual(get_added_content_summary(repo), {})
 
+    def test_rpm_kickstart(self):
+        """Sync repositories with the rpm plugin.
+
+        In order to sync a repository a remote has to be associated within
+        this repository. When a repository is created this version field is set
+        as None. After a sync the repository version is updated.
+
+        Do the following:
+
+        1. Create a repository and a remote.
+        2. Assert that repository version is None.
+        3. Sync the remote.
+        4. Assert that repository version is not None.
+        5. Assert that the correct number of units were added and are present
+           in the repo.
+        6. Sync the remote one more time.
+        7. Assert that repository version is different from the previous one.
+        8. Assert that the same number of are present and that no units were
+           added.
+        """
+        repo = self.client.post(REPO_PATH, gen_repo())
+        self.addCleanup(self.client.delete, repo['_href'])
+
+        # Create a remote with the standard test fixture url.
+        body = gen_rpm_remote(url=RPM_KICKSTART_FIXTURE_URL)
+        remote = self.client.post(RPM_REMOTE_PATH, body)
+        self.addCleanup(self.client.delete, remote['_href'])
+
+        # Sync the repository.
+        self.assertIsNone(repo['_latest_version_href'])
+        sync(self.cfg, remote, repo)
+        repo = self.client.get(repo['_href'])
+        import pdb; pdb.set_trace()
```

Comment:
```suggestion

```

Reply:
It still on development, and at the end of the day I push everything as is, just to send on scrum status